config.js

```diff
@@ -20,13 +20,20 @@ exports.create = (api) => {
       const keys = ssbKeys.loadOrCreateSync(Path.join(config.path, 'secret'))
       const pubkey = keys.id.slice(1).replace(`.${keys.curve}`, '')
 
-      config = merge(config, {
-        connections: {
-          incoming: { unix: [{ 'scope': 'local', 'transform': 'noauth' }] }
-        },
-        keys,
-        remote: `unix:${Path.join(config.path, 'socket')}:~noauth:${pubkey}`
-      })
+      if (process.platform !== "win32") {
+        config = merge(config, {
+          connections: {
+            incoming: { unix: [{ 'scope': 'local', 'transform': 'noauth' }] }
+          },
+          keys,
+          remote: `unix:${Path.join(config.path, 'socket')}:~noauth:${pubkey}`
+        })
+      } else {
+        config = merge(config, {
+          keys,
+          remote: `net:127.0.0.1:${config.port}~shs:${pubkey}`
```

Comment:
If we want to be IPv-neutral, we could also do:

```suggestion
          remote: `net:localhost:${config.port}~shs:${pubkey}`
```

:man_shrugging:

Reply:
@christianbundy see this comment in patchwork: https://github.com/ssbc/patchwork/blob/9433261107177d6178c2858447046b7b11991afc/index.js#L185

Reply:
😱 TIL Windows handles localhost differently thani remember, thanks!

Reply:
Exactly :D